### PR TITLE
Fix a grammar nit in CRYPTO_get_ex_new_index.pod

### PR DESCRIPTION
**Pull request by InfoHunter**
##### Checklist

- [x] documentation is added or updated

### CODE REVIEW

File: doc/man3/CRYPTO_get_ex_new_index.pod [hunk 1]
```diff
@@ -100,7 +100,7 @@ to avoid likely double-free crashes.
 The function B<CRYPTO_free_ex_data> is used to free all exdata attached
 to a structure. The appropriate type-specific routine must be used.
 The B<class_index> identifies the structure type, the B<obj> is
-be the pointer to the actual structure, and B<r> is a pointer to the
+the pointer to the actual structure, and B<r> is a pointer to the
```
Comment (paulidale) [inline doc/man3/CRYPTO_get_ex_new_index.pod]: I'd use _a_ not _the_ here.

**Comment by InfoHunter** [comment]
have replaced `the` with `a`.

**Comment by InfoHunter** [comment]
And this should be back ported to 1.1.1 as well

**Comment by InfoHunter** [comment]
Pushed to master (84712024da) and 1.1.1 (d600f3d34c). Thanks for reviewing ;-)

**Comment by levitte** [comment]
Nice catch, thank you

**Pull request closed without merging**